src/components/add-button/add-button.js

```diff
@@ -0,0 +1,11 @@
+import './add-button.css'
+
+function AddButton () {
```

Comment:
```suggestion
const AddButton = ()  => {
```
Better use arrow functions when possible. https://javascript.plainenglish.io/5-ways-to-optimize-your-functional-react-components-cb3cf6c7bd68